tests-ng/test_packages.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from plugins.dpkg import Dpkg
+from plugins.shell import ShellRunner
+
+denylist = [
+    "rlogin",
+    "rsh",
+    "rcp",
+    "telnet",
+    "libdb5.3:amd64",
+    "libdb5.3:arm64",
+    "libssl1.1:amd64",
+    "libssl1.1:arm64",
+]
+
+
+@pytest.mark.parametrize("denied_package", denylist)
+def test_no_denylisted_packages(denied_package: str, shell: ShellRunner):
+    dpkg = Dpkg(shell)
```

Comment:
Rather than importing and then creating a class here, dpkg could just be a fixture